app/assets/javascripts/pretty-text/engines/discourse-markdown/emoji.js

```diff
@@ -322,6 +322,8 @@ export function setup(helper) {
     opts.features.inlineEmoji = !!siteSettings.enable_inline_emoji_translation;
     opts.emojiSet = siteSettings.emoji_set || "";
     opts.customEmoji = state.customEmoji;
+    opts.enableEmojiCDN = siteSettings.external_emoji_url_enabled;
```

Comment:
I don't think we need 2 settings here, it just grows our json paylod for no huge benefit. A single `external_emoji_url` can work. If it is blank then just assume not using external.